Changes

```diff
@@ -0,0 +1,12 @@
+Revision history for Mojolicious-Plugin-ServerStatus
+
+0.03    2015-04-30
+    - improved documentation
+    - updated dist meta information
+    - removed unneeded dependencies
+
+0.02    2014-03-19
+    - improved query string handling
```

Comment:
I got this from [MetaCPAN's diff tool](https://metacpan.org/diff/file?target=FUKAI/Mojolicious-Plugin-ServerStatus-0.02/lib/Mojolicious/Plugin/ServerStatus.pm&source=FUKAI/Mojolicious-Plugin-ServerStatus-0.01/lib/Mojolicious/Plugin/ServerStatus.pm). Please change it if I got it wrong :)